tortoisebeacon/votes_encoder.go

```diff
@@ -0,0 +1,42 @@
+package tortoisebeacon
+
+import "github.com/bits-and-blooms/bitset"
+
+func (tb *TortoiseBeacon) encodeVotes(currentRound votesSetPair, firstRound firstRoundVotes) (votesBitVector []uint64) {
```

Comment:
does need to be a method of `TortoiseBeacon`?

Reply:
Assuming the current state — no. I left it as a method to be able to add logs easily if we need them.